frontend/public/locales/fr/status.json

```diff
@@ -123,7 +123,7 @@
         "date-of-birth-is-past": "La date de naissance doit être dans le passé",
         "date-of-birth-is-past-valid": "Date de naissance trop éloignée dans le passé",
         "date-of-birth-month-required": "La date de naissance doit inclure un mois",
-        "date-of-birth-valid": "Le jour doit être valide pour le mois et l'année choisi",
+        "date-of-birth-valid": "La date sélectionnée doit être une date calendaire valide.",
```

Comment:
This error message shouldn't differ from the rest of the application, unless a different error message was officially provided.